### ISSUE TITLE
ValueError: unknown url type

### ISSUE THREAD

**Issue by Rudloff** [rg3/youtube-dl#1308]
```
pierre@Rudloff:~$ youtube-dl --verbose http://tael67.free.fr/videos/
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://tael67.free.fr/videos/']
[debug] youtube-dl version 2013.08.23
[debug] Python version 2.7.3 - Linux-3.2.0-4-amd64-x86_64-with-debian-7.1
[debug] Proxy map: {}
WARNING: Falling back on generic information extractor.
[generic] : Downloading webpage
[generic] : Extracting information
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/bin/youtube-dl/__main__.py", line 18, in <module>
  File "/usr/bin/youtube-dl/youtube_dl/__init__.py", line 651, in main

  File "/usr/bin/youtube-dl/youtube_dl/__init__.py", line 635, in _real_main

  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 560, in download
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 327, in extract_info
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 410, in process_ie_result
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 357, in process_ie_result
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 535, in process_info
  File "/usr/bin/youtube-dl/youtube_dl/FileDownloader.py", line 429, in _do_download
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 393, in open
    protocol = req.get_type()
  File "/usr/lib/python2.7/urllib2.py", line 255, in get_type
    raise ValueError, "unknown url type: %s" % self.__original
ValueError: unknown url type: Camping.mp4
```

**Comment by isole** [comment]
This re-occurs with 2017.01.29

```
$ sudo youtube-dl -U
youtube-dl is up-to-date (2017.01.29)
.....
[youtube] aDPANvMhYpM: Downloading webpage
[youtube] aDPANvMhYpM: Downloading video info webpage
[youtube] aDPANvMhYpM: Extracting video information
[youtube] aDPANvMhYpM: Downloading js player en_US-vflV3n15C
ERROR: Signature extraction failed: Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 1057, in _decrypt_signature
    video_id, player_url, s
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 971, in _extract_signature_function
    errnote='Download of %s failed' % player_url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 529, in _download_webpage
    res = self._download_webpage_handle(url_or_request, video_id, note, errnote, fatal, encoding=encoding, data=data, headers=headers, query=query)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 436, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 416, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 2001, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 421, in open
    protocol = req.get_type()
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 283, in get_type
    raise ValueError, "unknown url type: %s" % self.__original
ValueError: unknown url type: /yts/jsbin/player-en_US-vflV3n15C/base.js
 (caused by ValueError(u'unknown url type: /yts/jsbin/player-en_US-vflV3n15C/base.js',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by youcanjive** [comment]
I am having the same problem. I'm sorry but my understanding of this is all
very limited, but up until yesterday it was all working perfectly. I have
only used the Windows executable file, I simply don't understand how to do
it any other way. I open a "cmd" window, drag youtube-dl.exe into it then
paste the youtube vid url and simply hit enter. It's worked beautifully up
until today. And yes I have made sure I'm using the latest version. Do I
need to do something differently now ? Again, my apologies for being so
basic with it. Any hep, in layman's terms, would be greatly appreciated.

**Comment by morsik** [comment]
I have the same problem. Using `youtube-dl-2017.01.10-1.fc25.noarch`.

**Comment by morsik** [comment]
I downloaded version from git. Looks like it works with this: https://github.com/rg3/youtube-dl/pull/11892

**Comment by isole** [comment]
> I downloaded version from git. Looks like it works with this: #11892 

Indeed. Updated to 2017.01.31 and it works. Sorry for the noise

**Comment by Hrxn** [comment]
> I am having the same problem. I'm sorry but my understanding of this is all
very limited, but up until yesterday it was all working perfectly. I have
only used the Windows executable file, I simply don't understand how to do
it any other way. I open a "cmd" window, drag youtube-dl.exe into it then
paste the youtube vid url and simply hit enter. It's worked beautifully up
until today. And yes I have made sure I'm using the latest version. Do I
need to do something differently now ? Again, my apologies for being so
basic with it. Any hep, in layman's terms, would be greatly appreciated.

@youcanjive No, it's still working as before, just make sure you're using the latest version of _youtube-dl.exe_

**Comment by uptownhr** [comment]
I'm on 2017.02.10 and still seeing this problem. 

```
[PornHub] ph57ed983c038dc: Downloading webpage
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/bin/youtube-dl/__main__.py", line 19, in <module>
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 452, in main
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 442, in _real_main
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1796, in download
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 707, in extract_info
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 753, in process_ie_result
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1440, in process_video_result
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1704, in process_info
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1646, in dl
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/common.py", line 353, in download
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/http.py", line 61, in real_download
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 2006, in urlopen
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 423, in open
    protocol = req.get_type()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 285, in get_type
    raise ValueError, "unknown url type: %s" % self.__original
ValueError: unknown url type: /*ec119a7a6ec8b845325fea2dc116b*/c41376f8a66316f72f/*f2bfd*/c9878980f5b8/*d71f6*/f2bfd/*c759b83f327df9*/f2d8dcac/*a24ebdfc510c9f1639de9ca11591e*/b4d422470153c8676c2fa14/*c9878980f5b8*/d71f6/*d71f6*/ec119a7a6ec8b845325fea2dc116b/*f2bfd*/e6c364bd71503570d5decc415e59fb/*f2d8dcac*/c7dcd4cfd4b312b/*f2d8dcac*/b206/*ec119a7a6ec8b845325fea2dc116b*/c759b83f327df9/*e6c364bd71503570d5decc415e59fb*/a23fe6249045/*b4d422470153c8676c2fa14*/a24ebdfc510c9f1639de9ca11591e
```
